03_how_ln_works.asciidoc

```diff
@@ -89,518 +93,526 @@ Also, in case you are not familiar with P2PKH addresses and the basic format and
 ====
 
-We will now see how such a penalty mechanism can be included to the above construction of the commitment transactions.
-Usually the commitment transaction has at least two outputs, one for each partner.
-As new addition, a channel partner will encumber their own output with a timelock and a revocation secret.
-The timelock prevents the owner of the output to spend it immediately once the commitment transaction was included in a block.
-The timelock is usually measured in blocktime and can be up to 2016.
-On average, 2016 blocks represent or last two weeks (assuming a blocktime of 10 minutes as in the Bitcoin network).
-Be aware, the exact number of blocks used for the timeout can be set to different values by different software.
-2016 is just the maximum value, but smaller values are possible.
-footnote:[Users likely prefer a longer timeout value for a larger channel capacity.]
-Within the timelock anyone who knows the revocation secret can spend the output even before the timelock is over.
-Alice and Bob know only one half each of the revocation secret.
-But if they share their half with the other channel partner, then the other partner knows the full secret.
-In order to update the balance and receive a signature from Bob, Alice will have to share her half of the revocation secret of the current commitment transaction with Bob.
-
-Obviously, for every new update of the channel balance new revocation secrets have to be created and saved. 
-As long as a channel remains open, all revocation secrets ever created for this specific channel need to be kept as they might be needed in the future.
-Luckily, the secrets are rather small and it is only the channel partners who need to keep them, not the entire network.
+Now that we understand _why_ a penalty mechanism is needed and how it will prevent cheating, let's see _how_ it works in detail.
+
+Usually, the commitment transaction has at least two outputs, paying each channel partner. We change this to add a _timelock delay_ and a _revocation secret_, to one of the payments. The timelock prevents the owner of the output from spending it immediately once the commitment transaction is included in a block. The revocation secret, allows either party to immediately spend that payment, bypassing the timelock.
+
+So, in our example, Bob holds a commitment transaction which pays Alice _immediately_, but his own payment is delayed and revocable. Alice also holds a commitment transaction, but hers is the opposite: it pays Bob immediately but her own payment is delayed and revocable.
+
+The two channel partners hold half of the revocation secret, so that neither one knows the whole secret. If they share their half, then the other channel partner has the full secret and can use it to exercise the revocation condition. When signing a new commitment transaction, they revoke the previous commitment by exchanging the signature for the revocation secret.
```

Comment:
can we really call it half a secrete? 

--> check with the BOLT3 revocation secret derrivation. --> yes because of addition.